FEM/rf_mfp_new.cpp

```diff
@@ -3473,7 +3474,13 @@ double MFPGetNodeValue(long node, const string& mfp_name, int phase_number)
             if ((*vec_var_names)[i] == "PRESSURE1")
                 arguments[0] = pcs->GetNodeValue(node, var_idx);
             else if ((*vec_var_names)[i] == "TEMPERATURE1")
-                arguments[1] = pcs->GetNodeValue(node, var_idx);
+            {
+                arguments[1] = pcs->getTemperatureUnit() ==
+                                       (FiniteElement::CELSIUS && for_output)
```

Comment:
why do we need „for_output“ here?

Reply:
I thought that MFPGetNodeValue is called twice outside of Output. Now with your comment, I realized that the temperature only be converted to the Celsius value during Output. Thanks!

Just change the source code.

Reply:
Just found that   „for_output“  is still needed for the calling outside of Output. I guess that MFPGetNodeValue was implemented only for output before, but it is now called in two functions for some restrict BC.

Change: replaced my previous commit with a commit with a correction in the ternary.